src/wp-includes/functions.wp-scripts.php

```diff
@@ -166,20 +167,23 @@ function wp_add_inline_script( $handle, $data, $position = 'after' ) {
  *                                    as a query string for cache busting purposes. If version is set to false, a version
  *                                    number is automatically added equal to current installed WordPress version.
  *                                    If set to null, no version is added.
- * @param bool             $in_footer Optional. Whether to enqueue the script before `</body>` instead of in the `<head>`.
- *                                    Default 'false'.
+ * @param array             $args     {
+ *      Optional. An array of additional script strategies. Default empty array.
+ *
+ *      @type boolean   $in_footer    Optional. Default false.
+ *      @type string    $strategy     Optional. Values blocking|defer|async. Default 'blocking'.
+ * }
  * @return bool Whether the script has been registered. True on success, false on failure.
  */
-function wp_register_script( $handle, $src, $deps = array(), $ver = false, $in_footer = false ) {
+function wp_register_script( $handle, $src, $deps = array(), $ver = false, $args = array() ) {
```

Comment:
there are a few places in core where we call this function that should probably be updated to the new parameter signature.

Reply:
@adamsilverstein I've updated applicable calls, there were not many.